styleguide/components/00-settings/_colors.scss

```diff
@@ -106,100 +106,23 @@ $tints: (
-    'link-text-decoration-hover': none,
-
-    'cta-link-background': color('blue', 3),
-    'cta-link-background-hover': color('white'),
-
-    'cta-link-border': 1px solid color('dark-gray', 3),
-    'cta-link-border-hover': 1px solid color('dark-gray', 3),
-
-    'cta-link-title-color': color('blue', 3),
-    'cta-link-title-color-hover': color('blue', 3),
-
-    'cta-link-document-color': color('blue', 3),
-    'cta-link-document-color-hover': color('blue'),
-
-    'cta-link-icon-border': 1px solid color('dark-gray', 3),
-
-    'field-description-background': color('dark-gray', -6),
-    'field-description-color': color('dark-gray'),
-
-    'field-color': color('dark-gray'),
-    'field-background': color('white'),
-    'field-border': 1px solid color('dark-gray', 3),
-    'field-placeholder': color('dark-gray', 3),
-
-    'fieldset-border': 1px solid color('dark-gray', -1),
-    'fieldset-color': color('blue'),
-
-    'field-error-background': color('red', -2),
-    'field-error-color': color('dark-gray'),
-    'field-error-border': color('red'),
-
-    'label-color': color('dark-gray'),
-    'label-optional-color': color('dark-gray'),
-
-    'button-color': color('white'),
-    'button-color-hover': color('white'),
-
-    'button-background': color('blue'),
-    'button-background-hover': color('blue', -2),
-
-    'button-border': none,
-    'button-border-hover': none,
-
-    'tag-color': color('white'),
-    'tag-color-hover': color('white'),
-
-    'tag-background': color('dark-gray'),
-    'tag-background-hover': color('dark-gray'),
-
-    'form-steps-timeline-color': color('blue'),
-    'form-steps-prefix-color': color('white'),
-
-    'form-steps-inactive-step-label': color('dark-gray'),
-    'form-steps-inactive-prefix-background': color('dark-gray'),
-    'form-steps-inactive-prefix-color': color('white'),
-
-    'form-steps-previous-step-label': color('green'),
-    'form-steps-previous-step-background': color('green'),
-
-    'form-steps-active-step-label': color('blue'),
-    'form-steps-active-prefix-background': color('blue'),
-
-    'hamburger-menu-background': color('blue', -2),
-    'hamburger-menu-overlay-color': rgba(44, 63, 72, .6),
-
-    'hamburger-menu-link-border': 1.5px solid color('blue', -2),
-    'hamburger-menu-link-border-active': 1.5px solid color('blue', -2),
-    'hamburger-menu-link-border-hover': 1.5px solid color('blue', -2),
-
-    'hamburger-menu-link-background': transparent,
-    'hamburger-menu-link-background-active': color('blue', -2),
-    'hamburger-menu-link-background-hover': color('blue', -2),
-
-    'hamburger-menu-link-color': color('white'),
-    'hamburger-menu-link-color-active': color('white'),
-    'hamburger-menu-link-color-hover': color('white'),
-
-    'island-border': 1px solid color('blue', -2),
+    // Required colors!
+    'color-primary': color('cyan'),
+    'color-secondary': color('dark-gray'),
+    'color-light': color('white'),
+    'color-dark': color('cyan', 3),
```

Comment:
These don't make much sense in dark themes.

Reply:
Working on the default variables :)

Reply:
@delrueba I've added some color palettes for now. Time will tell if these are enough. We can start development with these though.

Reply:
I know naming things is a b*tch, but let's get them right from the start.
What about foreground / background?

Reply:
I'd rather avoid names like that. Because we're not sure the "foreground" will be used as a real foreground... I've divided it up in 3 palettes now. This should be a good start.